src/datagen/java/blusunrize/immersiveengineering/data/Recipes.java

```diff
@@ -1219,6 +1231,7 @@ private void mineralMixes(@Nonnull Consumer<FinishedRecipe> out)
 				.setBackground(ForgeRegistries.BLOCKS.getKey(Blocks.NETHERRACK))
 				.build(out, toRL("mineral/mephitic_quarzite"));
 		MineralMixBuilder.builder(nether)
+				.addNetherSpoils()
 				.addOre(Blocks.GRAVEL, .6f)
 				.addOre(Blocks.ANCIENT_DEBRIS, .4f)
```

Comment:
Might be an idea to remove gravel here and give a high fail chance, so that this vein mostly digs spoil?

Reply:
That's fixed in the nether veins PR, so I'll leave it be until then.